### DISCUSSION TITLE
Traffic sign speed limit

### DISCUSSION THREAD

**Discussion by SF4User** [devemux86/cruiser#293] [Q&A] [answered]
I search, search and search . . . but I can't find the setting for this option anywhere 😳
In Germany there are (felt) every 50 mtr. these road signs . . . in Switzerland of all places, where there are high penalties,
rather less. A display, like this would be helpful, because you quickly miss a sign.
![speed limit](https://github.com/devemux86/cruiser/assets/132825580/c027c2b2-6215-4c14-9b49-311441c3d8e7)
Many thanks for the support.
PS: I'm use Mapsforge

**Comment by WolfgangZ42** [comment, marked as answer]
Look at this thread: [https://github.com/devemux86/cruiser/discussions/135](https://github.com/devemux86/cruiser/discussions/135)

**Reply by SF4User** [reply]
Vielen Dank! ist installiert und eine perfekte Alternative zum Cruiser-Display (, das in den Panelen auf der rechten Seite ) zu versteckt ist
